test/extended/builds/s2i_root.go

```diff
@@ -37,7 +37,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds] s2i build with a root user imag
 		Before(oc)
 		defer After(oc)
 
-		firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", image.LocationFor("registry.ci.openshift.org/ocp/4.7:test-build-roots2i"))
+		firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", image.LocationFor("registry.ci.openshift.org/ocp/4.8:test-build-roots2i"))
```

Comment:
So distilling https://github.com/openshift/origin/pull/26149/files#r632694287 for this line.

1) temporary step, change this to `firstArgString := fmt.Sprintf("%s~https://github.com/sclorg/nodejs-ex", "registry.ci.openshift.org/ocp/4.8:test-build-roots2i")`

2) if tests from this file are OK, we reach out to @soltysh or @smarterclayton and have them mirroregistry.ci.openshift.org/ocp/4.8:test-build-roots2i to quay.io/openshift/community-e2e-images

3) then we revert this line back to the way you have it now